GoogleForCreators/web-stories-wp
# Media: Generate and backfill BlurHash for media items

## Context



## Summary

As a follow-up to #9550 and #9647, we want to adopt BlurHash as progressive enhancement on top of baseColor to get even nicer previews when media is loading.

## Relevant Technical Choices

Blurhash can be expensive to generate, so only one is generated at a time and only after the upload is complete. 

## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/145074451-1360b1bb-91c9-44ef-a700-9327c4f39ea5.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Open editor.
2. Go 3P media. 
3. Click on image tab ( Unsplash ). 
4. See blurhashes as they load. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9685
